build-system/tasks/runtime-test.js

```diff
@@ -57,16 +57,33 @@ function getConfig() {
   return extend(obj, karmaConfig.default);
 }
 
-function getAdExtensions() {
-  const adNetworks = [];
+function getAdTypes() {
```

Comment:
adExtensions is a wrong name, they're not really AMP extensions.

I can actually hard code all google ad types here as well.